Datadog.Trace.sln

```diff
@@ -266,7 +262,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Samples.DatabaseHelper", "s
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "UpdateVendors", "tools\UpdateVendors\UpdateVendors.csproj", "{72FB583A-A1B0-4B5F-8658-617B100DCD8E}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SynchronizeInstaller", "tools\SynchronizeInstaller\SynchronizeInstaller.csproj", "{EBDDF6D7-59B8-4831-80C6-458C1ABBFC63}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PrepareRelease", "tools\PrepareRelease\PrepareRelease.csproj", "{EBDDF6D7-59B8-4831-80C6-458C1ABBFC63}"
```

Comment:
Going forwards please make big unrelated changes in a separate PR. Thanks!

Reply:
I can roll this back and commit it separately, I was originally looking to manage the content of this package from a single tool.

Reply:
I can still do this yet, but I'd like to open it up for review otherwise